packages/odyssey-react/src/components/FieldGroup/FieldGroup.module.scss

```diff
@@ -13,12 +13,12 @@
 .root {
   max-width: $max-line-length;
   margin-block-start: 0;
-  margin-block-end: $spacing-m;
+  margin-block-end: var(--SpaceEmM);
```

Comment:
I think `SpaceRemM` is intended as the theme value here. The local variable can probably be `SpaceMarginBlockEnd` or just `MarginBlockEnd`.